scss/components/_table-styles.scss

```diff
@@ -253,6 +253,84 @@ h3 + .simple-table {
   td:first-child {
     border-right: 1px solid $gray-medium;
     padding-left: u(4rem);
-    font-style: italic;
+     font-style: italic;
   }
 }
+
+// For custom_table_block in streamfactory
+
+.custom-table {
+  box-shadow: -8px 0px 9px -2px $gray-medium inset;
```

Comment:
Tiny thing, but if you set this to `-10px 0px 20px -10px #d6d7d9 inset` it will keep the box shadow from coming in on the top and bottom (just from the right).